sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 1.6.0 (Unreleased)
+
+### Features Added
+
+- Introduced support for `CommunicationTokenCredential` with `resource_endpoint`, `token_credential` and `scopes`, allowing an Entra user with a Teams license to use Teams Phone Extensibility features through the Azure Communication Services resource.
```

Comment:
we do not expose the credential in identity sdk, so maybe we can only mention in sdks we are exposing it, wdyt?

Reply:
removed it from here